lerobot/common/robots/config.py

```diff
@@ -26,6 +26,17 @@ class RobotConfig(draccus.ChoiceRegistry, abc.ABC):
     # Directory to store calibration file
     calibration_dir: Path | None = None
 
+    def __post_init__(self):
+        if hasattr(self, "cameras"):
+            cameras = self.cameras
+            if cameras:
+                for cam_name, cam_config in cameras.items():
+                    for attr in ["width", "height", "fps"]:
+                        if getattr(cam_config, attr) is None:
+                            raise ValueError(
+                                f"Camera config for '{cam_name}' has None value for required attribute '{attr}'"
+                            )
```

Comment:
```suggestion
        if hasattr(self, "cameras") and self.cameras:
            for name, config in self.cameras.items():
                for attr in ["width", "height", "fps"]:
                    if getattr(config, attr) is None:
                        raise ValueError(
                            f"Specifying '{attr}' is required for the camera to be used in a robot"
                            )
```
Also did you make sure this is run? We don't need to `super().__post_init__()` in the subclasses?

Reply:
Addressed in commit: https://github.com/huggingface/lerobot/pull/1108/commits/a36536dc078cf7f04ccb2b302cb458506d3634af